guix_studio/studiox.cpp

```diff
@@ -456,7 +456,7 @@ void CstudioxApp::OnAppAbout()
 
 void CstudioxApp::OnUsersGuide()
 {
-    ShellExecute(NULL, _T("open"), _T("https://aka.ms/azrtos-guix-studio-user-guide"), NULL, NULL, SW_SHOW);
+    ShellExecute(NULL, _T("open"), _T("https://github.com/eclipse-threadx/rtos-docs"), NULL, NULL, SW_SHOW);
```

Comment:
This should point to:
https://github.com/eclipse-threadx/rtos-docs/blob/main/rtos-docs/guix/about-guix-studio.md

Reply:
updated